spec/system/admin_kudo_delete_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe 'Kudo management', type: :system do
+  let!(:kudo) { create(:kudo) }
+
+  before do
+    admin = create(:admin)
+    login_as(admin, scope: :admin)
+    driven_by(:selenium_chrome_headless)
+  end
+
+  it 'kudos#delete?' do
+    visit 'admins/kudos'
+    click_button 'Delete'
+    page.driver.browser.switch_to.alert.accept
+    expect(page).to have_text('Kudo was deleted successfully')
```

Comment:
it would be nice to check if page have kudo (for ex. title) before delete, and not have it after delete